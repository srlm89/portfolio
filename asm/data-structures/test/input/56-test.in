ERASE actual/56-test.out
INSERT 9 0.363401 James Bond 17 - Golden Eye [1995]
PRINT_IN_ORDER actual/56-test.out
INSERT 55 3.760187 Corpse Bride   720p
INSERT 41 7.830382 Species   720p
REMOVE_ID 55
PRINT_IN_ORDER actual/56-test.out
INSERT 33 2.222902 Black Hawk Down   1080p
INSERT 56 0.147647 Batman The Dark Knight   720p
INSERT 52 3.632297 Diary.Of.A.Sex.Addict.2008.720p.BluRay.x264-FLHD
REMOVE_ID 52
PRINT_IN_ORDER actual/56-test.out
INSERT 59 4.206138 James Bond 20 - Die Another Day [2002] [720p]
INSERT 5 6.810232 Jumanji [720p]
INSERT 1 8.073352 Kung Fu Panda [2008]   720p
INSERT 60 7.360648 Good Night And Good Luck   720p
REMOVE_ID 60
REMOVE_ID 9
PRINT_IN_ORDER actual/56-test.out
INSERT 57 7.774024 Fantastic Four   1080p
INSERT 3 5.948767 Hulk 2 - The Incredible Hulk - 720p
INSERT 17 5.877120 Con Air   720p
INSERT 2 1.700275 Burn E [720p]
INSERT 43 0.459461 Alien vs Predator 1-2  720p
REMOVE_ID 17
REMOVE_ID 33
PRINT_IN_ORDER actual/56-test.out
INSERT 13 7.934979 Sex drive   720p
INSERT 44 4.791268 The Matrix [1999] [1080p]
INSERT 40 9.846857 Danny the Dog   1080p
INSERT 21 0.567426 Death Sentence   1080p
INSERT 16 8.611714 Monsters vs Aliens - Mutant Pumpkins from Outer Space [2009] [720p]
INSERT 58 2.637351 Fearless - 720p
REMOVE_ID 40
REMOVE_ID 2
REMOVE_ID 16
PRINT_IN_ORDER actual/56-test.out
INSERT 28 3.738976 Lust Caution   720p
INSERT 71 5.019271 Face.Off.1997.720p
INSERT 30 2.797023 Eragon 720p
INSERT 64 0.654042 Back.to.the.Future.Part.II.1989.720p.HDTV.x264
INSERT 11 0.043881 Underworld Series
INSERT 14 0.727277 Street Kings   720p
INSERT 62 7.536161 Dragon Wars   720p
REMOVE_ID 28
REMOVE_ID 21
REMOVE_ID 41
PRINT_IN_ORDER actual/56-test.out
INSERT 51 1.952348 IMAX Ring of Fire [720p] [Documenter]
INSERT 23 2.173324 The Empress and the Warriors [2008] [720p]
INSERT 78 2.691799 X-Men 1 - 1080p
INSERT 4 0.481002 One Missed Call   720p
INSERT 10 8.333389 Monsters vs Aliens [2009] [720p]
INSERT 36 2.969507 Apocalypto   720p
INSERT 19 4.350968 Hitman    720p
INSERT 69 5.971742 Lawrence of Arabia   1080p
REMOVE_ID 58
REMOVE_ID 3
REMOVE_ID 62
REMOVE_ID 59
PRINT_IN_ORDER actual/56-test.out
INSERT 6 5.664048 Orphan (2009)    1080p
INSERT 63 2.743440 Rocky 1 [1976]  [BluRay] [720p] [AC3]
INSERT 32 2.281315 Spiderman 1   1080p
INSERT 38 2.895437 Darko 720P
INSERT 53 2.153081 SAW III   720p
INSERT 94 1.631285 Fantastic Four   1080p
INSERT 84 1.216556 Fast and the Furious 4 - 1080p
INSERT 72 2.168024 The Mummy Tomb Of The Dragon Emperor   720p
INSERT 61 0.013921 911 Mysteries Demolition    720p
REMOVE_ID 30
REMOVE_ID 51
REMOVE_ID 44
REMOVE_ID 10
PRINT_IN_ORDER actual/56-test.out
INSERT 91 0.740936 Watchmen [720p]
INSERT 45 8.388597 Ratatouille [2007]
INSERT 81 5.863738 Monsters vs Aliens - Mutant Pumpkins from Outer Space [2009] [720p]
INSERT 92 7.241683 Orphan (2009)    1080p
INSERT 104 2.344982 Sixth Sense   720p
INSERT 15 2.350615 Lord Of The Ring 1 - Fellowship of the Ring (ExtEd) [1080p]
INSERT 99 3.250980 Eagle Eye    1080p
INSERT 42 0.019785 Pixar.Short.Films.Collection.Volume.1.2007.m-HD.x264-TlN50p
INSERT 68 9.392730 Toy.Story.2.HD.720p.Hebrew.Dubbed.Also.English.x264-ALiS_20
INSERT 95 4.724932 American Pie Book Of Love   720p
REMOVE_ID 95
REMOVE_ID 104
REMOVE_ID 45
REMOVE_ID 71
REMOVE_ID 61
PRINT_IN_ORDER actual/56-test.out
INSERT 54 1.408617 9 [720p]
INSERT 80 0.406205 X-Men Origin - Wolverine [2009] [1080p]
INSERT 8 8.355404 James Bond 19 - The World is not Enough [1999] [720p]
INSERT 74 0.566879 300 [2006]
INSERT 105 6.614365 Braveheart 720p
INSERT 20 7.285353 Juno 2007 720p
INSERT 86 6.001688 Terminator.1984.720p.BluRay.x264-ESiR
INSERT 27 4.395059 The Mummy Tomb Of The Dragon Emperor   720p
INSERT 82 1.015807 Oceans.Thirteen.2007.720p.HDDVD.x264-SiNNERS
INSERT 75 4.955064 A Bug's Life [720p]
INSERT 109 3.227919 Lord Of The Ring  - The Fellowship Of The Ring   720p
REMOVE_ID 109
REMOVE_ID 80
REMOVE_ID 94
REMOVE_ID 64
REMOVE_ID 23
PRINT_IN_ORDER actual/56-test.out
INSERT 50 0.300735 Punisher Warzone [720p]
INSERT 25 3.044687 The.Shawshank.Redemption.1994.1080p.BluRay.x264-CiNEFiLE
INSERT 90 4.019953 Seven Pounds    720p
INSERT 101 0.497733 Labor Pain   720p
INSERT 114 5.535547 Behind Enemy Line 2001   720p
INSERT 100 3.838630 Around The World In 80 Days [2004] [720p]
INSERT 121 0.018121 Danny the Dog   1080p
INSERT 24 4.868061 Batman Dark Knight   1080p
INSERT 49 3.119065 Batman Begins   1080p
INSERT 89 8.216369 Rambo 4 [2008] [720p]
INSERT 132 7.965220 Resident Evil Series    1080p
INSERT 113 4.492128 The Mummy Tomb Of The Dragon Emperor   720p
REMOVE_ID 1
REMOVE_ID 91
REMOVE_ID 132
REMOVE_ID 89
REMOVE_ID 32
REMOVE_ID 99
PRINT_IN_ORDER actual/56-test.out
INSERT 125 6.062092 Se7en [720p]
INSERT 22 8.511876 Ong.Bak.2.2008.BluRay.x264-BestHD   720p
INSERT 129 8.913693 Battlestar Galactica The Plan [720p]
INSERT 73 5.673142 The Butterfly Effect 3 Revelations 720p
INSERT 48 3.887411 Eragon 720p
INSERT 108 4.821803 Dumb and Dumber   720p
INSERT 88 0.012647 007 Series
INSERT 37 3.503995 PIXAR Movie
INSERT 135 9.983423 Constantine HDDVD 1080p DTS x264
INSERT 12 8.464456 Godzilla 720p
INSERT 7 1.251520 Fight.Club.1999.720p.BluRay.DTS-ES.x264-DON
INSERT 112 4.431051 Kung Fu Panda (BluRay 720p)
INSERT 93 5.834207 Superman Batman Public Enemies 2009 720p
REMOVE_ID 75
REMOVE_ID 78
REMOVE_ID 38
REMOVE_ID 42
REMOVE_ID 54
REMOVE_ID 86
PRINT_IN_ORDER actual/56-test.out
INSERT 124 8.421891 The Dark Knight [2008]   1080p
INSERT 118 2.567956 James Bond 19 - The World is not Enough [1999] [720p]
INSERT 85 1.772691 21   1080p
INSERT 137 1.652085 Bride Wars   720p
INSERT 31 9.534366 Bad Boy I [1995] [720p]
INSERT 47 8.721621 Starship Troopers 1  1080p
INSERT 110 7.390254 Van.Helsing  1080p
INSERT 98 5.120232 Fifty Dead Men Walking   1080p
INSERT 140 5.818162 Exiled   720p
INSERT 147 1.213784 Daredevil - 720p
INSERT 106 7.881718 SAW I   720p
INSERT 126 2.987671 007 Goldfinger [1964] [BluRay] [720p] [DTS]
INSERT 127 3.180112 Face.Off.1997.720p
INSERT 150 6.232448 Oceans.Thirteen.2007.720p.HDDVD.x264-SiNNERS
REMOVE_ID 56
REMOVE_ID 53
REMOVE_ID 15
REMOVE_ID 137
REMOVE_ID 113
REMOVE_ID 11
REMOVE_ID 84
PRINT_IN_ORDER actual/56-test.out
