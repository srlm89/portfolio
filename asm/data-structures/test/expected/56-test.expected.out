[ 9 0.363401 James Bond 17 - Golden Eye [1995] ]
[ 9 0.363401 James Bond 17 - Golden Eye [1995] ][ 41 7.830382 Species   720p ]
[ 9 0.363401 James Bond 17 - Golden Eye [1995] ][ 33 2.222902 Black Hawk Down   1080p ][ 41 7.830382 Species   720p ][ 56 0.147647 Batman The Dark Knight   720p ]
[ 1 8.073352 Kung Fu Panda [2008]   720p ][ 5 6.810232 Jumanji [720p] ][ 33 2.222902 Black Hawk Down   1080p ][ 41 7.830382 Species   720p ][ 56 0.147647 Batman The Dark Knight   720p ][ 59 4.206138 James Bond 20 - Die Another Day [2002] [720p] ]
[ 1 8.073352 Kung Fu Panda [2008]   720p ][ 2 1.700275 Burn E [720p] ][ 3 5.948767 Hulk 2 - The Incredible Hulk - 720p ][ 5 6.810232 Jumanji [720p] ][ 41 7.830382 Species   720p ][ 43 0.459461 Alien vs Predator 1-2  720p ][ 56 0.147647 Batman The Dark Knight   720p ][ 57 7.774024 Fantastic Four   1080p ][ 59 4.206138 James Bond 20 - Die Another Day [2002] [720p] ]
[ 1 8.073352 Kung Fu Panda [2008]   720p ][ 3 5.948767 Hulk 2 - The Incredible Hulk - 720p ][ 5 6.810232 Jumanji [720p] ][ 13 7.934979 Sex drive   720p ][ 21 0.567426 Death Sentence   1080p ][ 41 7.830382 Species   720p ][ 43 0.459461 Alien vs Predator 1-2  720p ][ 44 4.791268 The Matrix [1999] [1080p] ][ 56 0.147647 Batman The Dark Knight   720p ][ 57 7.774024 Fantastic Four   1080p ][ 58 2.637351 Fearless - 720p ][ 59 4.206138 James Bond 20 - Die Another Day [2002] [720p] ]
[ 1 8.073352 Kung Fu Panda [2008]   720p ][ 3 5.948767 Hulk 2 - The Incredible Hulk - 720p ][ 5 6.810232 Jumanji [720p] ][ 11 0.043881 Underworld Series ][ 13 7.934979 Sex drive   720p ][ 14 0.727277 Street Kings   720p ][ 30 2.797023 Eragon 720p ][ 43 0.459461 Alien vs Predator 1-2  720p ][ 44 4.791268 The Matrix [1999] [1080p] ][ 56 0.147647 Batman The Dark Knight   720p ][ 57 7.774024 Fantastic Four   1080p ][ 58 2.637351 Fearless - 720p ][ 59 4.206138 James Bond 20 - Die Another Day [2002] [720p] ][ 62 7.536161 Dragon Wars   720p ][ 64 0.654042 Back.to.the.Future.Part.II.1989.720p.HDTV.x264 ][ 71 5.019271 Face.Off.1997.720p ]
[ 1 8.073352 Kung Fu Panda [2008]   720p ][ 4 0.481002 One Missed Call   720p ][ 5 6.810232 Jumanji [720p] ][ 10 8.333389 Monsters vs Aliens [2009] [720p] ][ 11 0.043881 Underworld Series ][ 13 7.934979 Sex drive   720p ][ 14 0.727277 Street Kings   720p ][ 19 4.350968 Hitman    720p ][ 23 2.173324 The Empress and the Warriors [2008] [720p] ][ 30 2.797023 Eragon 720p ][ 36 2.969507 Apocalypto   720p ][ 43 0.459461 Alien vs Predator 1-2  720p ][ 44 4.791268 The Matrix [1999] [1080p] ][ 51 1.952348 IMAX Ring of Fire [720p] [Documenter] ][ 56 0.147647 Batman The Dark Knight   720p ][ 57 7.774024 Fantastic Four   1080p ][ 64 0.654042 Back.to.the.Future.Part.II.1989.720p.HDTV.x264 ][ 69 5.971742 Lawrence of Arabia   1080p ][ 71 5.019271 Face.Off.1997.720p ][ 78 2.691799 X-Men 1 - 1080p ]
[ 1 8.073352 Kung Fu Panda [2008]   720p ][ 4 0.481002 One Missed Call   720p ][ 5 6.810232 Jumanji [720p] ][ 6 5.664048 Orphan (2009)    1080p ][ 11 0.043881 Underworld Series ][ 13 7.934979 Sex drive   720p ][ 14 0.727277 Street Kings   720p ][ 19 4.350968 Hitman    720p ][ 23 2.173324 The Empress and the Warriors [2008] [720p] ][ 32 2.281315 Spiderman 1   1080p ][ 36 2.969507 Apocalypto   720p ][ 38 2.895437 Darko 720P ][ 43 0.459461 Alien vs Predator 1-2  720p ][ 53 2.153081 SAW III   720p ][ 56 0.147647 Batman The Dark Knight   720p ][ 57 7.774024 Fantastic Four   1080p ][ 61 0.013921 911 Mysteries Demolition    720p ][ 63 2.743440 Rocky 1 [1976]  [BluRay] [720p] [AC3] ][ 64 0.654042 Back.to.the.Future.Part.II.1989.720p.HDTV.x264 ][ 69 5.971742 Lawrence of Arabia   1080p ][ 71 5.019271 Face.Off.1997.720p ][ 72 2.168024 The Mummy Tomb Of The Dragon Emperor   720p ][ 78 2.691799 X-Men 1 - 1080p ][ 84 1.216556 Fast and the Furious 4 - 1080p ][ 94 1.631285 Fantastic Four   1080p ]
[ 1 8.073352 Kung Fu Panda [2008]   720p ][ 4 0.481002 One Missed Call   720p ][ 5 6.810232 Jumanji [720p] ][ 6 5.664048 Orphan (2009)    1080p ][ 11 0.043881 Underworld Series ][ 13 7.934979 Sex drive   720p ][ 14 0.727277 Street Kings   720p ][ 15 2.350615 Lord Of The Ring 1 - Fellowship of the Ring (ExtEd) [1080p] ][ 19 4.350968 Hitman    720p ][ 23 2.173324 The Empress and the Warriors [2008] [720p] ][ 32 2.281315 Spiderman 1   1080p ][ 36 2.969507 Apocalypto   720p ][ 38 2.895437 Darko 720P ][ 42 0.019785 Pixar.Short.Films.Collection.Volume.1.2007.m-HD.x264-TlN50p ][ 43 0.459461 Alien vs Predator 1-2  720p ][ 53 2.153081 SAW III   720p ][ 56 0.147647 Batman The Dark Knight   720p ][ 57 7.774024 Fantastic Four   1080p ][ 63 2.743440 Rocky 1 [1976]  [BluRay] [720p] [AC3] ][ 64 0.654042 Back.to.the.Future.Part.II.1989.720p.HDTV.x264 ][ 68 9.392730 Toy.Story.2.HD.720p.Hebrew.Dubbed.Also.English.x264-ALiS_20 ][ 69 5.971742 Lawrence of Arabia   1080p ][ 72 2.168024 The Mummy Tomb Of The Dragon Emperor   720p ][ 78 2.691799 X-Men 1 - 1080p ][ 81 5.863738 Monsters vs Aliens - Mutant Pumpkins from Outer Space [2009] [720p] ][ 84 1.216556 Fast and the Furious 4 - 1080p ][ 91 0.740936 Watchmen [720p] ][ 92 7.241683 Orphan (2009)    1080p ][ 94 1.631285 Fantastic Four   1080p ][ 99 3.250980 Eagle Eye    1080p ]
[ 1 8.073352 Kung Fu Panda [2008]   720p ][ 4 0.481002 One Missed Call   720p ][ 5 6.810232 Jumanji [720p] ][ 6 5.664048 Orphan (2009)    1080p ][ 8 8.355404 James Bond 19 - The World is not Enough [1999] [720p] ][ 11 0.043881 Underworld Series ][ 13 7.934979 Sex drive   720p ][ 14 0.727277 Street Kings   720p ][ 15 2.350615 Lord Of The Ring 1 - Fellowship of the Ring (ExtEd) [1080p] ][ 19 4.350968 Hitman    720p ][ 20 7.285353 Juno 2007 720p ][ 27 4.395059 The Mummy Tomb Of The Dragon Emperor   720p ][ 32 2.281315 Spiderman 1   1080p ][ 36 2.969507 Apocalypto   720p ][ 38 2.895437 Darko 720P ][ 42 0.019785 Pixar.Short.Films.Collection.Volume.1.2007.m-HD.x264-TlN50p ][ 43 0.459461 Alien vs Predator 1-2  720p ][ 53 2.153081 SAW III   720p ][ 54 1.408617 9 [720p] ][ 56 0.147647 Batman The Dark Knight   720p ][ 57 7.774024 Fantastic Four   1080p ][ 63 2.743440 Rocky 1 [1976]  [BluRay] [720p] [AC3] ][ 68 9.392730 Toy.Story.2.HD.720p.Hebrew.Dubbed.Also.English.x264-ALiS_20 ][ 69 5.971742 Lawrence of Arabia   1080p ][ 72 2.168024 The Mummy Tomb Of The Dragon Emperor   720p ][ 74 0.566879 300 [2006] ][ 75 4.955064 A Bug's Life [720p] ][ 78 2.691799 X-Men 1 - 1080p ][ 81 5.863738 Monsters vs Aliens - Mutant Pumpkins from Outer Space [2009] [720p] ][ 82 1.015807 Oceans.Thirteen.2007.720p.HDDVD.x264-SiNNERS ][ 84 1.216556 Fast and the Furious 4 - 1080p ][ 86 6.001688 Terminator.1984.720p.BluRay.x264-ESiR ][ 91 0.740936 Watchmen [720p] ][ 92 7.241683 Orphan (2009)    1080p ][ 99 3.250980 Eagle Eye    1080p ][ 105 6.614365 Braveheart 720p ]
[ 4 0.481002 One Missed Call   720p ][ 5 6.810232 Jumanji [720p] ][ 6 5.664048 Orphan (2009)    1080p ][ 8 8.355404 James Bond 19 - The World is not Enough [1999] [720p] ][ 11 0.043881 Underworld Series ][ 13 7.934979 Sex drive   720p ][ 14 0.727277 Street Kings   720p ][ 15 2.350615 Lord Of The Ring 1 - Fellowship of the Ring (ExtEd) [1080p] ][ 19 4.350968 Hitman    720p ][ 20 7.285353 Juno 2007 720p ][ 24 4.868061 Batman Dark Knight   1080p ][ 25 3.044687 The.Shawshank.Redemption.1994.1080p.BluRay.x264-CiNEFiLE ][ 27 4.395059 The Mummy Tomb Of The Dragon Emperor   720p ][ 36 2.969507 Apocalypto   720p ][ 38 2.895437 Darko 720P ][ 42 0.019785 Pixar.Short.Films.Collection.Volume.1.2007.m-HD.x264-TlN50p ][ 43 0.459461 Alien vs Predator 1-2  720p ][ 49 3.119065 Batman Begins   1080p ][ 50 0.300735 Punisher Warzone [720p] ][ 53 2.153081 SAW III   720p ][ 54 1.408617 9 [720p] ][ 56 0.147647 Batman The Dark Knight   720p ][ 57 7.774024 Fantastic Four   1080p ][ 63 2.743440 Rocky 1 [1976]  [BluRay] [720p] [AC3] ][ 68 9.392730 Toy.Story.2.HD.720p.Hebrew.Dubbed.Also.English.x264-ALiS_20 ][ 69 5.971742 Lawrence of Arabia   1080p ][ 72 2.168024 The Mummy Tomb Of The Dragon Emperor   720p ][ 74 0.566879 300 [2006] ][ 75 4.955064 A Bug's Life [720p] ][ 78 2.691799 X-Men 1 - 1080p ][ 81 5.863738 Monsters vs Aliens - Mutant Pumpkins from Outer Space [2009] [720p] ][ 82 1.015807 Oceans.Thirteen.2007.720p.HDDVD.x264-SiNNERS ][ 84 1.216556 Fast and the Furious 4 - 1080p ][ 86 6.001688 Terminator.1984.720p.BluRay.x264-ESiR ][ 90 4.019953 Seven Pounds    720p ][ 92 7.241683 Orphan (2009)    1080p ][ 100 3.838630 Around The World In 80 Days [2004] [720p] ][ 101 0.497733 Labor Pain   720p ][ 105 6.614365 Braveheart 720p ][ 113 4.492128 The Mummy Tomb Of The Dragon Emperor   720p ][ 114 5.535547 Behind Enemy Line 2001   720p ][ 121 0.018121 Danny the Dog   1080p ]
[ 4 0.481002 One Missed Call   720p ][ 5 6.810232 Jumanji [720p] ][ 6 5.664048 Orphan (2009)    1080p ][ 7 1.251520 Fight.Club.1999.720p.BluRay.DTS-ES.x264-DON ][ 8 8.355404 James Bond 19 - The World is not Enough [1999] [720p] ][ 11 0.043881 Underworld Series ][ 12 8.464456 Godzilla 720p ][ 13 7.934979 Sex drive   720p ][ 14 0.727277 Street Kings   720p ][ 15 2.350615 Lord Of The Ring 1 - Fellowship of the Ring (ExtEd) [1080p] ][ 19 4.350968 Hitman    720p ][ 20 7.285353 Juno 2007 720p ][ 22 8.511876 Ong.Bak.2.2008.BluRay.x264-BestHD   720p ][ 24 4.868061 Batman Dark Knight   1080p ][ 25 3.044687 The.Shawshank.Redemption.1994.1080p.BluRay.x264-CiNEFiLE ][ 27 4.395059 The Mummy Tomb Of The Dragon Emperor   720p ][ 36 2.969507 Apocalypto   720p ][ 37 3.503995 PIXAR Movie ][ 43 0.459461 Alien vs Predator 1-2  720p ][ 48 3.887411 Eragon 720p ][ 49 3.119065 Batman Begins   1080p ][ 50 0.300735 Punisher Warzone [720p] ][ 53 2.153081 SAW III   720p ][ 56 0.147647 Batman The Dark Knight   720p ][ 57 7.774024 Fantastic Four   1080p ][ 63 2.743440 Rocky 1 [1976]  [BluRay] [720p] [AC3] ][ 68 9.392730 Toy.Story.2.HD.720p.Hebrew.Dubbed.Also.English.x264-ALiS_20 ][ 69 5.971742 Lawrence of Arabia   1080p ][ 72 2.168024 The Mummy Tomb Of The Dragon Emperor   720p ][ 73 5.673142 The Butterfly Effect 3 Revelations 720p ][ 74 0.566879 300 [2006] ][ 81 5.863738 Monsters vs Aliens - Mutant Pumpkins from Outer Space [2009] [720p] ][ 82 1.015807 Oceans.Thirteen.2007.720p.HDDVD.x264-SiNNERS ][ 84 1.216556 Fast and the Furious 4 - 1080p ][ 88 0.012647 007 Series ][ 90 4.019953 Seven Pounds    720p ][ 92 7.241683 Orphan (2009)    1080p ][ 93 5.834207 Superman Batman Public Enemies 2009 720p ][ 100 3.838630 Around The World In 80 Days [2004] [720p] ][ 101 0.497733 Labor Pain   720p ][ 105 6.614365 Braveheart 720p ][ 108 4.821803 Dumb and Dumber   720p ][ 112 4.431051 Kung Fu Panda (BluRay 720p) ][ 113 4.492128 The Mummy Tomb Of The Dragon Emperor   720p ][ 114 5.535547 Behind Enemy Line 2001   720p ][ 121 0.018121 Danny the Dog   1080p ][ 125 6.062092 Se7en [720p] ][ 129 8.913693 Battlestar Galactica The Plan [720p] ][ 135 9.983423 Constantine HDDVD 1080p DTS x264 ]
[ 4 0.481002 One Missed Call   720p ][ 5 6.810232 Jumanji [720p] ][ 6 5.664048 Orphan (2009)    1080p ][ 7 1.251520 Fight.Club.1999.720p.BluRay.DTS-ES.x264-DON ][ 8 8.355404 James Bond 19 - The World is not Enough [1999] [720p] ][ 12 8.464456 Godzilla 720p ][ 13 7.934979 Sex drive   720p ][ 14 0.727277 Street Kings   720p ][ 19 4.350968 Hitman    720p ][ 20 7.285353 Juno 2007 720p ][ 22 8.511876 Ong.Bak.2.2008.BluRay.x264-BestHD   720p ][ 24 4.868061 Batman Dark Knight   1080p ][ 25 3.044687 The.Shawshank.Redemption.1994.1080p.BluRay.x264-CiNEFiLE ][ 27 4.395059 The Mummy Tomb Of The Dragon Emperor   720p ][ 31 9.534366 Bad Boy I [1995] [720p] ][ 36 2.969507 Apocalypto   720p ][ 37 3.503995 PIXAR Movie ][ 43 0.459461 Alien vs Predator 1-2  720p ][ 47 8.721621 Starship Troopers 1  1080p ][ 48 3.887411 Eragon 720p ][ 49 3.119065 Batman Begins   1080p ][ 50 0.300735 Punisher Warzone [720p] ][ 57 7.774024 Fantastic Four   1080p ][ 63 2.743440 Rocky 1 [1976]  [BluRay] [720p] [AC3] ][ 68 9.392730 Toy.Story.2.HD.720p.Hebrew.Dubbed.Also.English.x264-ALiS_20 ][ 69 5.971742 Lawrence of Arabia   1080p ][ 72 2.168024 The Mummy Tomb Of The Dragon Emperor   720p ][ 73 5.673142 The Butterfly Effect 3 Revelations 720p ][ 74 0.566879 300 [2006] ][ 81 5.863738 Monsters vs Aliens - Mutant Pumpkins from Outer Space [2009] [720p] ][ 82 1.015807 Oceans.Thirteen.2007.720p.HDDVD.x264-SiNNERS ][ 85 1.772691 21   1080p ][ 88 0.012647 007 Series ][ 90 4.019953 Seven Pounds    720p ][ 92 7.241683 Orphan (2009)    1080p ][ 93 5.834207 Superman Batman Public Enemies 2009 720p ][ 98 5.120232 Fifty Dead Men Walking   1080p ][ 100 3.838630 Around The World In 80 Days [2004] [720p] ][ 101 0.497733 Labor Pain   720p ][ 105 6.614365 Braveheart 720p ][ 106 7.881718 SAW I   720p ][ 108 4.821803 Dumb and Dumber   720p ][ 110 7.390254 Van.Helsing  1080p ][ 112 4.431051 Kung Fu Panda (BluRay 720p) ][ 114 5.535547 Behind Enemy Line 2001   720p ][ 118 2.567956 James Bond 19 - The World is not Enough [1999] [720p] ][ 121 0.018121 Danny the Dog   1080p ][ 124 8.421891 The Dark Knight [2008]   1080p ][ 125 6.062092 Se7en [720p] ][ 126 2.987671 007 Goldfinger [1964] [BluRay] [720p] [DTS] ][ 127 3.180112 Face.Off.1997.720p ][ 129 8.913693 Battlestar Galactica The Plan [720p] ][ 135 9.983423 Constantine HDDVD 1080p DTS x264 ][ 140 5.818162 Exiled   720p ][ 147 1.213784 Daredevil - 720p ][ 150 6.232448 Oceans.Thirteen.2007.720p.HDDVD.x264-SiNNERS ]
